google-cloud-core/src/main/java/com/google/cloud/HttpTransportOptions.java

```diff
@@ -154,10 +156,27 @@ public void initialize(HttpRequest httpRequest) throws IOException {
         if (readTimeout >= 0) {
           httpRequest.setReadTimeout(readTimeout);
         }
+
+        HttpHeaders headers = httpRequest.getHeaders();
+        headers.set("x-goog-api-client", getXGoogApiClientHeader());
       }
     };
   }
 
+  String getXGoogApiClientHeader() {
+    return String.format(
+        "gl-java/%s %s/%s gax/%s",
```

Comment:
The gax version isn't relevant here either, right? The equivalent would be the version of google-api-java-client I think. cc @bjwatson